### ISSUE TITLE
can not get selected option in Multipleselect in many to many relation

### ISSUE THREAD

**Issue by AhmedHdeawy** [z-song/laravel-admin#1917]
- Laravel Version: 5.5
- PHP Version: 7
- Laravel-admin: 1.5.*

### Description:
first of all, thanks for this awesome package

i can't get selected values tags in edit article

my form() method like
`$form->multipleSelect('tags', 'Tags')->options(Tag::all()->pluck('name', 'id'));`

and in Article Model, the relation method like
`public function tags(){
    return $this->belongsToMany('App\Tag')->select('id','name')->withTimestamps();
  }`

when i press edit button for any article, some fields working fine like 'title', 'description' but input tags field doesn't show selected tags for this article, just show all tags which come from Tag::all(), the input tags selected empty

how i can solve this proplem?

some screenshots

![1](https://user-images.githubusercontent.com/15928500/38776977-55256e10-40a0-11e8-9db9-ae245589445d.png)

![2](https://user-images.githubusercontent.com/15928500/38776980-5dffb612-40a0-11e8-9793-ab76799f0d8e.png)

![3](https://user-images.githubusercontent.com/15928500/38776984-62a1f95a-40a0-11e8-8904-a4ab8ef6536a.png)

**Comment by mcraz** [comment]
I had problems similar problems. I moved to explicit declaration and it works. Like so:
```
$form
    ->multipleSelect('tags', 'Tags')
    ->ajax('/api/list/tags')
    ->options(function() { return $this->tags->pluck('name', 'id'); });
```

**Comment by jimmitjoo** [comment]
I have this exact same problem.

**Comment by z-song** [comment]
Fix in #2479

**Comment by binhlvit** [comment]
I have this exact same problem.
